### PR TITLE
Lower ATEN ones and zeros to XLA broadcasted scalar ops.

### PR DESCRIPTION
**Pull request by dlibenzi**

### CODE REVIEW

File: torch_xla/csrc/aten_xla_bridge.h [hunk 1]
```diff
@@ -14,7 +15,10 @@ namespace bridge {
 
 // Extracts the XLATensor out of our version of at::Tensor. Throws an exception
 // if tensor is not an XLA tensor.
-XLATensor& GetXlaTensor(const at::Tensor& tensor);
+XLATensor GetXlaTensor(const at::Tensor& tensor);
+
+// Fetches the underline XLATensor object if tensor is an XLA tensor.
+c10::optional<XLATensor> TryGetXlaTensor(const at::Tensor& tensor);
```
Comment (asuhan) [inline torch_xla/csrc/aten_xla_bridge.h]: Why is this in the header? It's a helper only used in `torch_xla/csrc/aten_xla_bridge.cpp` as far as I can tell.

Comment (dlibenzi) [reply torch_xla/csrc/aten_xla_bridge.h]: I can see other parts of the codebase wanting a do-this-if-xla-tensor-else-...

Comment (asuhan) [reply torch_xla/csrc/aten_xla_bridge.h]: It's very easy to move something out of an anonymous namespace and add its signature to a header. All else being equal, smaller interfaces are nicer.

Comment (dlibenzi) [reply torch_xla/csrc/aten_xla_bridge.h]: Alrighty.

**Pull request merged**